packages/worker/src/index.ts

```diff
@@ -8,13 +8,13 @@
  * @module
  */
 
-export { State, Worker, DataConverter, defaultPayloadConverter, errors } from './worker';
-export { WorkerOptions, CompiledWorkerOptions } from './worker-options';
-export { NativeConnectionOptions, RequiredNativeConnectionOptions, TLSConfig } from './connection-options';
-export { Runtime, RuntimeOptions, TelemetryOptions, History } from './runtime';
 export { NativeConnection as NativeConnection } from './connection';
+export { NativeConnectionOptions, RequiredNativeConnectionOptions, TLSConfig } from './connection-options';
+export * from './errors';
+export * from './interceptors';
 export * from './logger';
+export { History, Runtime, RuntimeOptions, TelemetryOptions } from './runtime';
 export * from './sinks';
-export * from './interceptors';
-export { bundleWorkflowCode, BundleOptions } from './workflow/bundler';
-export * from './errors';
+export { DataConverter, defaultPayloadConverter, errors, State, Worker } from './worker';
+export { CompiledWorkerOptions, WorkerOptions } from './worker-options';
+export { BundleOptions, bundleWorkflowCode } from './workflow/bundler';
```

Comment:
Shouldn't this also export `nodejsBuiltinModules` or am I missing something?

Reply:
Intended usage is:

```ts
import { nodejsBuiltinModules } from '@temporalio/worker/lib/workflow/bundler'
```

Would export from index if wanted users to import it, or to include in API reference